src/styles/utils/_variables.scss

```
@@ -0,0 +1,13 @@
+$dark-text-color: #060b35;
+$blue-color: #00acdc;
+$grey-color: #f3f3f3;
+$light-grey: #616070;
+$white-color: #fff;
+$header-logo-size: 40px;
+$number-of-cards-in-a-row: 4;
+$header-height: 60px;
+$card-width: 200px;
+$card-margin-right: 50px;
+$card-margin-bottom: 48px;
```

the distance between cards on the design includes borders (zoom in to see it);
```suggestion
$card-margin-right: 48px;
$card-margin-bottom: 46px;
```

---

OMG! You are right! That's why i had a mistake... +